demosplan/DemosPlanCoreBundle/Logic/Import/Statement/StatementSpreadsheetImporter.php

```diff
@@ -176,15 +177,24 @@ public function process(SplFileInfo $workbook): void
             $originalStatementOrViolations = $builder->buildStatementAndReset();
             if ($originalStatementOrViolations instanceof Statement) {
                 $externId = $originalStatementOrViolations->getExternId();
+                $internId = $originalStatementOrViolations->getInternId(false);
                 if (array_key_exists($externId, $usedExternIds)) {
                     // skip statements with existing extern IDs
                     $this->skippedStatements[$externId] = ($this->skippedStatements[$externId] ?? 0) + 1;
 
                     continue;
                 }
+                if (array_key_exists($internId, $usedInternIds)) {
```

Comment:
internId can be null, therefore imports with more than one STN without (null) an internId, only the first STN without an internId will be imported.

Reply:
fix in [4e52b43](https://github.com/demos-europe/demosplan-core/pull/2575/commits/8586a664b1fd40f8f48c3106c1f375ee1eb30ef8)